HW_1/Ex_8/Ex_8.py

```diff
@@ -0,0 +1,52 @@
+# Задача 8: Требуется определить, можно ли от шоколадки размером n
+# × m долек отломить k долек, если разрешается сделать один разлом по
+# прямой между дольками (то есть разломить шоколадку на два
+# прямоугольника).
+
+
+
+# ___________________РЕШЕНИЕ ЗДОРОВОГО ЧЕЛОВЕКА___________________________
+
+n = int(input("Введите длинну (n) шоколадки: "))
+m = int(input("Введите ширину (m) шоколадки: "))
+
+
+print("Какого размера кусочек вы хотите отломить?")
+piece = int(input("Введите: "))
```

Comment:
Это можно совместить, ведь это запрос к пользователю
```suggestion
piece = int(input("Какого размера кусочек вы хотите отломить?\nВведите: "))
```